src/components/EditSection.tsx

```diff
@@ -11,6 +11,13 @@ interface EditSectionProps {
 
 const AVATARS_TO_DISPLAY = 8
 
+const GENERATED_PAGE_DATA = {
+    'company/team/index.md': 'team.yml',
+    'product/feature_maturity.md': 'product_areas.yml',
+    'product/product_areas.md': 'features.yml',
```

Comment:
This was renamed in main

```suggestion
    'product/product_teams.md': 'product_teams.yml',
```